src/main/java/com/gojek/beast/sink/bq/handler/DefaultLogWriter.java

```diff
@@ -24,7 +24,7 @@ public Status writeErrorRecords(List<Record> records) {
             records.forEach(record -> {
                 log.debug("Error record: {} columns: {}", record.getId(), record.getColumns());
             });
-            statsClient.gauge("sink.logwriter.invalid.records", records.size());
+            statsClient.gauge("data.error.records,type=oob", records.size());
```

Comment:
Let us capture this metric in the OOBErrorHandler itself, adding it here means that we need to add this metric in all writers. Lets add it the same way we are adding metric for counting invalid and unknown messages